cli/internal/run/run_test.go

```diff
@@ -40,8 +40,10 @@ func TestParseConfig(t *testing.T) {
 					Dir:     defaultCacheFolder,
 					Workers: 10,
 				},
-				runcacheOpts: runcache.Opts{},
-				scopeOpts:    scope.Opts{},
+				runcacheOpts: runcache.Opts{
+					TaskOutputMode: util.FullTaskOutput,
```

Comment:
In Go, whenever possible, we try to make the zero-value for a variable or field in a struct meaningful. You can see this in practice with `CacheHitLogsMode` and `CacheMissLogsMode` in [this struct](https://github.com/vercel/turborepo/blob/main/cli/internal/runcache/runcache.go#L44). This is using Go's version of an `enum`, which is defined [here](https://github.com/vercel/turborepo/blob/main/cli/internal/runcache/runcache.go#L29). `iota` maps to 0, so that when a `LogsMode` is not specified, we default to `FullLogs`. 

I think we want the same to apply to this new field. That likely means following a similar pattern of using an enum and translating the string inputs to enum values.

That will let us avoid all the changes setting a default value of `FullTaskOutput`, as that would be the zero value for the new field.